src/main/java/seedu/address/logic/commands/sale/AllListCommand.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.logic.commands.sale;
```

Comment:
@hakujitsu file renamed to AllListCommand for both show all and for specified contact. 

changes are that it extends from abstract class ListCommand which stores some static variables and a common method (formatting sale list) that was abstracted from your previous class so that both our classes can use it.

On top of that, message strings are set as static variables to be consistent with other command design. Do you let me know what you think!